vspace/src/main/java/edu/asu/diging/vspace/web/exception/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,79 @@
+    protected ModelAndView handleModuleNotFoundException(HttpServletRequest request, ModuleNotFoundException ex) {
+        ModelAndView modelAndView = new ModelAndView();
+        modelAndView.addObject("error", ex.getMessage());
+        logger.info("ModuleNotFoundException Occured:: URL=" + request.getRequestURL());
+        modelAndView.addObject("url", request.getRequestURL());
+        modelAndView.setViewName("module");
+        return modelAndView;
+    }
+
+    @ExceptionHandler({ ModuleNotConfiguredException.class })
+    protected ModelAndView handleModuleNotConfiguredException(HttpServletRequest request,
+            ModuleNotConfiguredException ex) {
+        ModelAndView modelAndView = new ModelAndView();
+        modelAndView.addObject("error", ex.getMessage());
+        logger.info("ModuleNotConfiguredException Occured:: URL=" + request.getRequestURL());
+        modelAndView.addObject("url", request.getRequestURL());
+        modelAndView.setViewName("module");
+        return modelAndView;
+    }
+
+    @ExceptionHandler({ SlideNotFoundException.class })
+    protected ModelAndView handleSlideNotFoundException(HttpServletRequest request, SlideNotFoundException ex) {
+        ModelAndView modelAndView = new ModelAndView();
+        modelAndView.addObject("error", ex.getMessage());
+        logger.info("SlideNotFoundException Occured:: URL=" + request.getRequestURL());
+        modelAndView.addObject("url", request.getRequestURL());
+        modelAndView.setViewName("module");
+        return modelAndView;
+    }
+
+    @ExceptionHandler({ SequenceNotFoundException.class })
+    protected ModelAndView handleSequenceNotFoundException(HttpServletRequest request, SequenceNotFoundException ex) {
+        ModelAndView modelAndView = new ModelAndView();
+        modelAndView.addObject("error", ex.getMessage());
+        logger.info("SequenceNotFoundException Occured:: URL=" + request.getRequestURL());
+        modelAndView.addObject("url", request.getRequestURL());
+        modelAndView.setViewName("module");
+        return modelAndView;
+    }
+
+    @ExceptionHandler({ SlidesInSequenceNotFoundException.class })
+    protected ModelAndView handleSlidesInSequenceNotFoundException(HttpServletRequest request,
+            SlidesInSequenceNotFoundException ex) {
+        ModelAndView modelAndView = new ModelAndView();
+        modelAndView.addObject("error", ex.getMessage());
+        logger.info("SlidesInSequenceNotFoundException Occured:: URL=" + request.getRequestURL());
+        modelAndView.addObject("url", request.getRequestURL());
+        modelAndView.setViewName("module");
+        return modelAndView;
+    }
+
+    @ExceptionHandler({ SpaceNotFoundException.class })
+    protected ModelAndView handleSpaceNotFoundException(HttpServletRequest request,
+            SlidesInSequenceNotFoundException ex) {
+        ModelAndView modelAndView = new ModelAndView();
+        modelAndView.addObject("error", ex.getMessage());
```

Comment:
since these will be internationalized eventually, I wouldn't use the message from the exception here.